vms/proposervm/pre_fork_block.go

```diff
@@ -79,10 +101,6 @@ func (b *preForkBlock) verifyPreForkChild(ctx context.Context, child *preForkBlo
 			return err
 		}
 
-		if err := b.verifyIsPreForkBlock(); err != nil {
```

Comment:
can you clarify why these checks has been removed? A pre-fork block happening post fork would pass here. Could you comment on what engine invariant guarantees that despite not checking here, such a pre-fork block would be rejected?

Reply:
Because we are verifying the child block - we can assume that the parent block was successfully verified. This wasn't the case previously because the parent block could have returned as `Accepted` previously (which the engine assumes implies it was already verified).